packages/babel-preset-gatsby/src/utils/path-serializer.ts

```diff
@@ -0,0 +1,19 @@
+import { slash } from "gatsby-core-utils"
+
+const cleanNodeModules = (dir: string): string => {
+  const x = dir.split(`node_modules/`)
+
+  if (x.length === 0) {
```

Comment:
This change seems to cause problems with tests:

```suggestion
  if (x.length <= 1) {
```

Reply:
Read it wrongly, so will revert 🙂